### DISCUSSION TITLE
Hotkey mute or unmute for specific application

### DISCUSSION THREAD

**Discussion by SupernovaCooper** [File-New-Project/EarTrumpet#870] [Ideas]
Hi,Thanks for this greate software.

Would you like to add a feature for it please? When I playing games, and I also watching a movie during waiting time, but I need to change the volume channel by myself, so could you add a feature which is control the volume or mute or unmute for specific appliaction by the hotkey ? Then I can just press the hotkey to control the game application. It's will be greate. 

For example, on the volume channel window, right click on each application, offer some options to set hotkeys.

Thank you.

**Comment by riverar** [comment]
@SupernovaCooper Hey there! If you download the [dev builds of EarTrumpet](https://eartrumpet.app/), you'll find Eartrumpet Actions in Settings. This will let you build hotkeys and other macros. Give it a try.

**Reply by SupernovaCooper** [reply]
Hi, I just tried, it's wonderful ! thank you !!!

**Reply by kanjieater** [reply]
Any update on when muting or unmuting actions would be available in the main build?

**Reply by riverar** [reply]
Actions still has some rough edges but I'm actively working on the project now to get some things moving. Stay tuned and thanks for your patence!

**Reply by rwxOsu** [reply]
thank you, this is exactly what I needed, and it works perfectly

**Comment by Arxareon** [comment]
Any chance of this being included as a mainline feature for the stable release soon?

(Not full macro capability and everything, just quick mute/unmute of the focused window/application. I saw multiple feature requests for this.)
